### ISSUE TITLE
 .NET Core compatible version

### ISSUE THREAD

**Issue by NimmiW** [digipost/digipost-api-client-dotnet#136]
Is There a dot net core compatible version of this Nuget Package? I am currently trying to integrate this to a dotnet core 2.x project. 

Thank you in advance for any information or clarification :)

**Comment by NimmiW** [comment]
I am getting the following error when I used Gigipost-api-client in a .net core project.

The type 'IDataType' is defined in an assembly that is not referenced. You must add a reference to assembly 'Digipost.Api.Client.DataTypes, 
Version=8.2.0.0, Culture=neutral, PublicKeyToken=75878787'.

**Comment by asjafjell** [comment]
Hi, and thank you for your inquiry. We are trying to make a .NET Standard version of the libraries, but are currently blocked by an issue with sending encrypted HTTP-requests, so it has been postponed. W e do not know when we will have time to look into it again. The problem must be fixed within .NET core first and added to .NET standard.

**Comment by Ovini** [comment]
I've used the library with .netcore 2.1 and didn't give any issues.
